pkg/vulnerabilityreport/controller/scanjob.go

```diff
@@ -357,6 +357,10 @@ func (r *ScanJobController) completedContainers(ctx context.Context, scanJob *ba
 }
 
 func (r *ScanJobController) deleteJob(ctx context.Context, job *batchv1.Job) error {
+	if job.Spec.TTLSecondsAfterFinished != nil && *job.Spec.TTLSecondsAfterFinished != 0 {
```

Comment:
I read through the docs but it wasn't clear to me why we should check for both `!=nil` and `!=0`. Could you explain that?

Reply:
Actually I just realiezed that `!=0` is incorrect. If `ttl` is 0, the job could obviously be deleted. I would remove that from the if statement.

Reply:
Actually now that you mentioned it, it seems to me that the `!=0` is superfluous / not really necessary. I would remove this from the if statement.

Reply:
Could you update your PR?

Reply:
@simar7 I'm sorry but somehow I totally forgot to do the update. I made the change and squashed the commits so that the commit history looks better.